src/pages/api/invite-user.ts

```diff
@@ -66,13 +85,157 @@ export const POST: APIRoute = async ({ request, cookies }) => {
 
   const body = await request.json();
 
-  const inviteResp = await supa.auth.admin.inviteUserByEmail(body.email);
+  // Create the user and then send an immediate
+  const inviteResp = await supa.auth.admin.createUser({
+    email: body.email,
+    password: generatePassword(14),
+  });
 
   if (inviteResp.error) {
-    return new Response(JSON.stringify({ error: 'Failed to invite user' }), {
+    return new Response(JSON.stringify({ error: 'Failed to create user' }), {
       status: 500,
     });
   }
 
+  const i18n = getDefaultTranslations('email');
+
+  // create a token
+  const key = Buffer.from(INVITE_CRYPTO_KEY, 'base64');
+  const token = encrypt(`${inviteResp.data.user.id}|${Date.now()}`, key);
+
+  const { t, lang } = i18n;
+
+  const config: TReaderDocument = {
+    root: {
+      type: 'EmailLayout',
+      data: {
+        backdropColor: '#b5c8dc',
+        borderRadius: 8,
+        canvasColor: '#FFFFFF',
+        textColor: '#242424',
+        fontFamily: 'MODERN_SANS',
+        childrenIds: [
+          'block-1709571212684',
+          'block-1709571228545',
+          'block-1709571234315',
+          'block-1709571302968',
+        ],
+      },
+    },
+    'block-1709571212684': {
+      type: 'Image',
+      data: {
+        style: {
+          padding: {
+            top: 24,
+            bottom: 24,
+            right: 24,
+            left: 24,
+          },
+        },
+        props: {
+          url: `${url.protocol}//${url.host}/img/branding/email/top-logo.svg`,
+          alt: 'RecogitoStudio',
+          linkHref: url.host,
+          contentAlignment: 'middle',
+        },
+      },
+    },
+    'block-1709571228545': {
+      type: 'Text',
+      data: {
+        style: {
+          fontWeight: 'normal',
+          padding: {
+            top: 0,
+            bottom: 16,
+            right: 24,
+            left: 24,
+          },
+        },
+        props: {
+          text: t['Greetings'],
+        },
+      },
+    },
+    'block-1709571234315': {
+      type: 'Text',
+      data: {
+        style: {
+          fontWeight: 'normal',
+          padding: {
+            top: 0,
+            bottom: 16,
+            right: 24,
+            left: 24,
+          },
+        },
+        props: {
+          text: t['_join_recogito_message_'].replaceAll(
```

Comment:
Changed that to `replaceAll` since I ended up with a DE version that uses the instance name twice.